pkgs/by-name/bi/bitbox-bridge/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  libudev-zero
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bitbox-bridge";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "BitBoxSwiss";
+    repo = "bitbox-bridge";
+    tag = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-+pMXWXGHyyBx3N0kiro9NS0mPmSQzzBmp+pkoBLH7z0=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-6vD0XjGH1PXjiRjgnHWSZSixXOc2Yecui8U5FAGefBU=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libudev-zero
+  ];
+
+  doCheck = false;
+
+  meta = {
+    description = "A bridge service that connects web wallets like Rabbit to BitBox02";
+    homepage = "https://github.com/BitBoxSwiss/bitbox-bridge";
+    downloadPage = "https://bitbox.swiss/download/";
+    changelog = "https://github.com/BitBoxSwiss/bitbox-bridge/blob/master/CHANGELOG.md#${
```

Comment:
Probably best to just do ${version} and replace masters here instead of doing all of this extra work.